readme.md

```diff
@@ -312,50 +211,43 @@ yarn closure
 yarn compile-devtools
 ```
 
-## Lighthouse as trace processor
-
-Lighthouse can be used to analyze trace and performance data collected from other tools (like WebPageTest and ChromeDriver). The `traces` and `devtoolsLogs` artifact items can be provided using a string for the absolute path on disk. The devtoolsLogs array is captured from the Network domain (a la ChromeDriver's [`enableNetwork` option](https://sites.google.com/a/chromium.org/chromedriver/capabilities#TOC-perfLoggingPrefs-object)) and reformatted slightly. As an example, here's a trace-only run that's reporting on user timings and critical request chains:
-
-### `config.json`
-
-```json
-{
-  "audits": [
-    "user-timings",
-    "critical-request-chains"
-  ],
-
-  "artifacts": {
-    "traces": {
-      "defaultPass": "/User/me/lighthouse/lighthouse-core/test/fixtures/traces/trace-user-timings.json"
-    },
-    "devtoolsLogs": {
-      "defaultPass": "/User/me/lighthouse/lighthouse-core/test/fixtures/traces/perflog.json"
-    }
-  },
-
-  "aggregations": [{
-    "name": "Performance Metrics",
-    "description": "These encapsulate your app's performance.",
-    "scored": false,
-    "categorizable": false,
-    "items": [{
-      "audits": {
-        "user-timings": { "expectedValue": 0, "weight": 1 },
-        "critical-request-chains": { "expectedValue": 0, "weight": 1}
-      }
-    }]
-  }]
-}
-```
+## Docs & Recipes
+
+Useful documentation, examples, and recipes to get you started.
+
+**Docs**
```

Comment:
should this be moved higher? Not as useful for first time visitors, but useful for repeat visitors. At least, when I need to get to a particular set of instructions I usually go to a project's readme and hope that it's linked to from there. Don't know what other people do :)